test/e2e/query_test.go

```diff
@@ -1720,6 +1721,38 @@ func rangeQuery(t *testing.T, ctx context.Context, addr string, q func() string,
 	return retExplanation
 }
 
+func RemoteWrite(ctx context.Context, timeseries []prompb.TimeSeries, addr string) error {
```

Comment:
Change it to a non public function

Reply:
done

Reply:
replica label is needed (giving an error otherwise), I think that was attached by the Prometheus instance